src/adv.py

```diff
@@ -1,51 +1,180 @@
-to north. The smell of gold permeates the air."""),
-
-    'treasure': Room("Treasure Chamber", """You've found the long-lost treasure
-chamber! Sadly, it has already been completely emptied by
-earlier adventurers. The only exit is to the south."""),
+    'lumbridge': Room("Lumbridge", """Welcome to the kingdom of Lumbridge!""",[]),
+    'entrance': Room("Entrance", """Please come in!""",['grass','rocks', 'dust']),
+    'weaponstore': Room("Weapon Store", """Variety of weapons to choose from.""",['sword', 'dagger', 'knife']),
+    'bank': Room("Bank", """Access your personal bank here""",['100 gbp']),
+    'library': Room("Library", """All information about Lumbridge is available here. From our people to weapons.""",['Modern History', 'Mountain Trolls', 'Varrock City']),
+    'dine': Room("Dining Hall", """Incredible feast are freshly prepared everyday""",['Cake', 'Baguette', 'Meatloaf'])
 }
 
-
-# Link rooms together
-
-room['outside'].n_to = room['foyer']
-room['foyer'].s_to = room['outside']
-room['foyer'].n_to = room['overlook']
-room['foyer'].e_to = room['narrow']
-room['overlook'].s_to = room['foyer']
-room['narrow'].w_to = room['foyer']
-room['narrow'].n_to = room['treasure']
-room['treasure'].s_to = room['narrow']
+# # Link rooms together
+room['lumbridge'].n_to = room['entrance']
+room['entrance'].s_to = room['lumbridge']
+room['entrance'].n_to = room['weaponstore']
+room['entrance'].w_to = room['bank']
+room['entrance'].e_to = room['library']
+room['weaponstore'].s_to = room['entrance']
+room['bank'].w_to = room['entrance']
+room['bank'].n_to = room['library']
+room['library'].s_to = room['bank']
 
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
 
+player = Player("Bob", room['lumbridge'], ['WOOD', 'LIGHTER', 'TUNA'])
+clear()
+welcome_msg = """
+ ──────▄▀▀▀▀▀▀▀▄─────── 
+ ─────▐─▄█▀▀▀█▄─▌────── 
+ ─────▐─▀█▄▄▄█▀─▌────── 
+ ──────▀▄▄▄▄▄▄▄▀─────── 
+ ─────▐▀▄▄▐█▌▄▄▀▌────── 
+ ──────▀▄▄███▄▄▀─────── 
+ █▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀█ 
+ █░░╦─╦╔╗╦─╔╗╔╗╔╦╗╔╗░░█ 
+ █░░║║║╠─║─║─║║║║║╠─░░█ 
+ █░░╚╩╝╚╝╚╝╚╝╚╝╩─╩╚╝░░█ 
+ █▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄█ 
+ """
+print(welcome_msg)
+
 # Write a loop that:
 #
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
+# * Prints the current room name/description
+print("\nWelcome " + player.name + "! \nWe are very excited for you to begin your quest. \n")
+
+print("You are currently logged into the world: " + player.current_room.name + ".\n" + player.current_room.description + "\n\n")
+
+def room_items():
+    output = ''
+    for i in player.current_room.room_items:
+        output += (" " + i + "\n✿ ❀ ❁ ✾ ✽ ❃\n\n")
+    return output
+
+def add_items():
+    item = input("If you would like to take an item, type 'Take' and the item. \n Or you may type 'No': ")
+    if len(item) != 0:
+        if item.split()[0].upper() == 'TAKE' or item.split()[0].upper() == 'NO':
```

Comment:
i would make `item.split()[0].upper()` it's own variable. something descriptive like `userInput`